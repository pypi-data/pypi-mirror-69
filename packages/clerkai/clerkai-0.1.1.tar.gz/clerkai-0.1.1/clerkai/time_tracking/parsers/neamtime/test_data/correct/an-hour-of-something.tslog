start 2018-04-14 14:00ca

15:00ca, foo
paus->
