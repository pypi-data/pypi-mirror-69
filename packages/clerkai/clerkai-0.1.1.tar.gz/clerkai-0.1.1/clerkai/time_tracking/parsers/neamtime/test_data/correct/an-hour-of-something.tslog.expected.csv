Raw Ignore,Raw Source Line Numbers,Raw Source Lines Summary,Raw Session,Raw UTC Timestamp,Raw Work Date,Raw Timezone,Raw Time-annotated Source Lines Summary,Raw Hours,Raw Annotation,Raw Year,Raw Year-half,Raw Month,Raw Week,Raw Client,Raw Invoice,Raw Invoice row ordinal,Raw Invoice row item,Raw Invoice row item category,Raw Invoice row item reference,Raw Invoice hours,Ignore,Source Line Numbers,Source Lines Summary,Session,UTC Timestamp,Work Date,Timezone,Time-annotated Source Lines Summary,Hours,Annotation,Year,Year-half,Month,Week,Client,Invoice,Invoice row ordinal,Invoice row item,Invoice row item category,Invoice row item reference,Invoice hours,Original data
,, foo,start 2018-04-14 14:00ca,2018-04-14 15:00,2018-04-14,UTC, 1h0min foo,1,Uncategorized,,,,,,,,,,,,,,foo,start 2018-04-14 14:00ca,2018-04-14 15:00:00,2018-04-14 00:00:00,UTC,1h0min foo,1,Uncategorized,,,,,,,,,,,,"{'gmtTimestamp': '2018-04-14 15:00', 'category': 'Uncategorized', 'date': '2018-04-14', 'dateRaw': '\t2018-04-14 15:00', 'hours': 1, 'hoursRounded': 1, 'lineWithoutDate': ' 1h0min foo', 'text': ' foo', 'ts': 1523718000, 'tz': 'UTC', 'sessionMeta.session_ref': 'start 2018-04-14 14:00ca', 'sessionMeta.tzFirst': 'UTC'}"
